common-util/Login/LoginV2.jsx

```diff
@@ -152,35 +147,23 @@ export const LoginV2 = ({
 
   return (
     <LoginContainer>
-      <Web3NetworkSwitch />
+      <w3m-network-button />
       &nbsp;&nbsp;
-      <Web3Button
+      <w3m-button
         avatar="hide"
```

Comment:
don't see avatar and icon properties for this element, are you sure they work? https://docs.walletconnect.com/web3modal/react/components#w3m-button-

Reply:
good catch will remove unused props